src/Microsoft.TestPlatform.Extensions.BlameDataCollector/PublicAPI/PublicAPI.Shipped.txt

```diff
@@ -58,7 +58,7 @@ Microsoft.TestPlatform.Extensions.BlameDataCollector.IProcessDumpUtility
 Microsoft.TestPlatform.Extensions.BlameDataCollector.IProcessDumpUtility.DetachFromTargetProcess(int targetProcessId) -> void
 Microsoft.TestPlatform.Extensions.BlameDataCollector.IProcessDumpUtility.GetDumpFiles(bool warnOnNoDumpFiles, bool processCrashed) -> System.Collections.Generic.IEnumerable<string>
 Microsoft.TestPlatform.Extensions.BlameDataCollector.IProcessDumpUtility.StartHangBasedProcessDump(int processId, string testResultsDirectory, bool isFullDump, string targetFramework, System.Action<string> logWarning = null) -> void
-Microsoft.TestPlatform.Extensions.BlameDataCollector.IProcessDumpUtility.StartTriggerBasedProcessDump(int processId, string testResultsDirectory, bool isFullDump, string targetFramework, bool collectAlways) -> void
+Microsoft.TestPlatform.Extensions.BlameDataCollector.IProcessDumpUtility.StartTriggerBasedProcessDump(int processId, string testResultsDirectory, bool isFullDump, string targetFramework, bool collectAlways, System.Action<string> logWarning) -> void
```

Comment:
Safe to do, I did it few times in the past as well.. This should not even be public api.